### DISCUSSION TITLE
Any interest in updates to Qt5+?

### DISCUSSION THREAD

**Discussion by jsprenkle** [rodlie/qtfm#179] [General]
I just downloaded the new Qt and noticed this project uses deprecated Qt packages (QRegExp etc).
Any interest in having the updates to the newer version of Qt?

**Comment by rodlie** [comment]
Hi,

I guess you mean Qt6? I'm in no rush to do anything about that, I assume things still work.

If someone want to update qtfm for Qt6 feel free to submit a PR, just don't break compat with the Qt versions we currently support.

**Comment by jsprenkle** [comment]
They removed deprecated classes, like QRegExp, that is used in the code.
Not sure when QRegularExpression was added.

The QConcurrent calls are not compiling either.

You might be able to be 5 and 6 compatible. but I'm thinking compatibility
will be difficult.

On Fri, Oct 8, 2021 at 4:16 AM Ole-André ***@***.***> wrote:

> Hi,
>
> I guess you mean Qt6? I'm in no rush to do anything about that, I assume
> things still work.
>
> If someone want to update qtfm for Qt6 feel free to submit a PR, just
> don't break compat with the Qt versions we currently support.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rodlie/qtfm/discussions/179#discussioncomment-1447175>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADIKGIWI4NN5ZPRGSK4OYDUF2ZFPANCNFSM5FMH5JTA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 

A study recently showed <http://time.com/3858309/attention-spans-goldfish/>
that the average human attention span is now shorter than that of goldfish.
--
  Technology trends, nerd humor, and prognostications about the future
science:   Facebook <https://www.facebook.com/JaySprenkle>  | Twitter
<https://twitter.com/jsprenkle1>
  My personal technology blog <https://jaysprenkle.wordpress.com/>
